app/controllers/people_controller.rb

```diff
@@ -0,0 +1,38 @@
+class PeopleController < ApplicationController
```

Comment:
I'm not sure about the indentation of this file - I think you want the indentation to be the same between the class and method definitions, and the def keyword and the method contents.